src/Path.php

```diff
@@ -190,7 +190,7 @@ public static function check($path, $basePath = '')
      */
     public static function clean($path, $ds = \DIRECTORY_SEPARATOR)
     {
-        if (!\is_string($path)) {
+        if (!\is_string($path) && !empty($path)) {
             throw new \InvalidArgumentException('You must specify a non-empty path to clean');
         }
```

Comment:
Using an early return makes it easier to understand
```suggestion
        if ($path === '') {
            return '';
        }

        if (!\is_string($path)) {
            throw new \InvalidArgumentException('You must specify a non-empty path to clean');
        }
```